### ISSUE TITLE
Starlink 9 Payloads Orbit info null

### ISSUE THREAD

**Issue by waruss321** [r-spacex/SpaceX-API#454]
No payload orbit info for the Starlink-9 mission. Payload ID: 5ed9858b1f30554030d45c3e 

Just noticed that I wasn't getting any data back even though the mission has been completed. 

`
"longitude": null,
"semi_major_axis_km": null,
"eccentricity": null,
"periapsis_km": null,
"apoapsis_km": null,
"inclination_deg": null,
"period_min": null,
"lifespan_years": null,
"epoch": null
`

**Comment by jakewmeyer** [comment]
Yeah, the Norad id's took a bit to come out for that, and the TLE epoch was out of the range the script looks at.

Should be resolved now 👍

**Comment by waruss321** [comment]
Awesome! Thank you!
